apps/rig_tests/test/proxy/response_from/http_test.exs

```diff
@@ -1,15 +1,21 @@
 defmodule RigTests.Proxy.ResponseFrom.HttpTest do
   @moduledoc """
   If `response_from` is not set, the response is taken from the proxied request.
+  If `response_from` is set to http_internal, the response is taken from internal HTTP endpoint /v1/responses
```

Comment:
similar to how we did it for the Kafka response_from target, I'd prefer having a dedicated test module for this feature